packages/fast-components-react-base/src/dialog/dialog.props.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { IDialogClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface IDialogHandledProps {
+    /**
+     * The dialog content
+     */
+    children?: React.ReactNode | React.ReactNode[];
+
+    /**
+     * The dialog content width
+     */
+    contentWidth?: string;
+
+    /**
+     * The dialog content height
+     */
+    contentHeight?: string;
+
+    /**
+     * The aria-describedby attribute to link the dialog to an
+     * element that describes its purpose
+     */
+    describedBy?: string;
+
+    /**
+     * The aria-label to provide an accessible name for the dialog
+     */
+    label?: string;
+
+    /**
+     * The aria-labelledby attribute to link the dialog to an existing
+     * element that provides it an accessible name
+     */
+    labelledBy?: string;
+
+    /**
+     * The option to add modal functionality and prevent a user
+     * from interacting with elements outside the dialog
+     */
+    modal?: boolean;
+
+    /**
+     * Sets the visibility of the dialog to assistive technologies
```

Comment:
just to assistive technologies? doesn't it also set visibility of the modal visually?

Reply:
In base, just to assistive technologies. More in other comment.